### PR TITLE
Cu 82ruhy build darwin binary

### PR DESCRIPTION
**Pull request by julien-leclercq**
Build binary for macos...

### CODE REVIEW

**Comment by kdisneur** [comment]
If we add the vendoring (and use `go run ./vendor/....`) it will fix your test issue :)

**Comment by kdisneur** [comment]
By the way in addition to the issue you're fixing, I also have permissions issues.

After a `brew install fewlinesco/tap/fwl_error` I get

```
zsh: permission denied: fwl-error
```

and if I look at the permissions I don't have the `x` right

```
ls -l /usr/local/Cellar/fwl_error/1.0.2/bin/fwl-error
-r--r--r--  1 kdisneur  staff  2183768 Sep 17 10:06 /usr/local/Cellar/fwl_error/1.0.2/bin/fwl-error
```

**Comment by julien-leclercq** [comment]
> By the way in addition to the issue you're fixing, I also have permissions issues.
> 
> After a `brew install fewlinesco/tap/fwl_error` I get
> 
> ```
> zsh: permission denied: fwl-error
> ```
> 
> and if I look at the permissions I don't have the `x` right
> 
> ```
> ls -l /usr/local/Cellar/fwl_error/1.0.2/bin/fwl-error
> -r--r--r--  1 kdisneur  staff  2183768 Sep 17 10:06 /usr/local/Cellar/fwl_error/1.0.2/bin/fwl-error
> ```

Yes I know but isn't it a problem with homebrew ? I tried to fix it yesterday, without success ...

**Comment by julien-leclercq** [comment]
Should I just chmod the binary before pushing it ?

**Review by kdisneur — DISMISSED** [review]

**Review by kdisneur — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -17,8 +17,8 @@ MANIFEST_PATH=manifest.txt
 
 GO_BIN := go
 GO_FMT_BIN := gofmt
-GO_LINT := golint
-GO_STATICCHECK_BIN := staticcheck
+GO_LINT := go run ./vendor/golang.org/x/lint/golint
```
Comment (kdisneur) [inline Makefile:20]:
```suggestion
GO_LINT := $(GO_BIN) run ./vendor/golang.org/x/lint/golint
```

**Pull request merged**